cea/optimization/preprocessing/decentralized_buildings.py

```diff
@@ -246,10 +252,9 @@ def calc_new_load(mdot, TsupDH, Tret, gv):
         # Check the GHP area constraint
         for i in range(10):
             QGHP = (1-i/10) * Qnom
-            areaAvail = geothermal_potential.ix[building_name,"Area_geo"]
+            areaAvail = geothermal_potential.ix[geothermal_potential['Name'] == building_name, 'Area_geo']
```

Comment:
I suggest setting the index of the geothermal_potential to "Name" and then use the formulation geothermal_potnetial.ix[building_name, "Area_geo].